lib/tasks/user.rake

```diff
@@ -0,0 +1,8 @@
+namespace :user do
+  desc 'Creating default categories for all current users'
+  task generate_categories: :environment do
```

Comment:
@marderer what do you think about renaming it the task to `generate_default_categories`? Cause it's the thing tasks do